### ISSUE TITLE
Rule "ignoreFiles" does not exist

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#441]
Hey!

I have the following config:

`.linthtmlrc.js`:
```js
const commonConfig = require('./.builder/.linthtmlrc.json');

module.exports = {
  ...commonConfig,
};
```

`.builder/.linthtmlrc.json`:
```json
{
  "attr-req-value": false,
  "id-class-style": false,
  "img-req-alt": "allownull",
  "indent-style": "spaces",
  "indent-width": 2,
  "lang-style": false,
  "raw-ignore-regex": "{{.*?}}",
  "title-max-len": false,
  "no-surrounding-whitespace": true
}
```

Up to 0.8.3 it worked fine. Since 0.8.4, this error is thrown:

```
Rule "ignoreFiles" does not exist
Rule "ignoreFiles" does not exist
```

Thanks, as always, for your great job :)

**Comment by KamiKillertO** [comment]
🤔 that's weird. Thanks for reporting a will look into it

**Comment by KamiKillertO** [comment]
@acalvo I've released a fix :+

**Comment by acalvo** [comment]
Thanks! Can't try it right now, will try to do it ASAP. Thanks again!
